mutable_tree.go

```diff
@@ -138,7 +138,7 @@ func (tree *MutableTree) Get(key []byte) ([]byte, error) {
 	}
 
 	if !tree.skipFastStorageUpgrade {
-		if fastNode, ok := tree.unsavedFastNodeAdditions[ibytes.UnsafeBytesToStr(key)]; ok {
+		if fastNode, ok := tree.unsavedFastNodeAdditions[string(key)]; ok {
```

Comment:
is it because the string conversion is the same as the unsafe version in go 1.20? or because the performance is impact illegible.

Reply:
map[string([]byte)] is a no-copy, no garbage operation in Go since some releases ago.

It's the same performance, but safe.